Policies/dmca-takedown-policy.md

```diff
@@ -54,7 +54,7 @@ Here are the basic steps in the process.
 
 ### B. What About Forks? (or What's a Fork?)
 
-One of the best features of [Git](https://git-scm.com/book/en/Getting-Started-Git-Basics) is the ability for users to "fork" one another's repositories. What does that mean? In essence, it means that users can make a copy of a project and then make changes to that copy to either push back to the main project or just keep as their own variation of a project. We call each of these copies a "[fork](/articles/github-glossary#fork)" of the original repository, which in turn may also be called the "parent" of the fork.
+One of the best features of GitHub is the ability for users to "fork" one another's repositories. What does that mean? In essence, it means that users can make a copy of a project on GitHub into their own repositories. As the license or the law allows, Users can then make changes to that fork to either push back to the main project or just keep as their own variation of a project. Each of these copies is a "[fork](/articles/github-glossary#fork)" of the original repository, which in turn may also be called the "parent" of the fork.
```

Comment:
I think you want to keep “users” (which you're currently changing to “Users”), because it is not defined locally in this document:

```
$ git grep Users origin/pr/65 --  Policies/dmca-takedown-policy.md
origin/pr/65:Policies/dmca-takedown-policy.md:One of the best features of GitHub is the ability for users to "fork" one another's repositories. What does that mean? In essence, it means that users can make a copy of a project on GitHub into their own repositories. As the license or the law allows, Users can then make changes to that fork to either push back to the main project or just keep as their own variation of a project. Each of these copies is a "[fork](/articles/github-glossary#fork)" of the original repository, which in turn may also be called the "parent" of the fork.
```